ci/test_notebooks.sh

```diff
@@ -1,17 +1,24 @@
 #!/bin/bash
-# Copyright (c) 2020-2024, NVIDIA CORPORATION.
+# Copyright (c) 2020-2025, NVIDIA CORPORATION.
 
 set -Eeuo pipefail
 
 . /opt/conda/etc/profile.d/conda.sh
 
 RAPIDS_VERSION="$(rapids-version)"
 
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
+
 rapids-logger "Generate notebook testing dependencies"
 rapids-dependency-file-generator \
   --output conda \
   --file-key test_notebooks \
-  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}"  \
```

Comment:
```suggestion
  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}"  \
  --prepend-channel dglteam/label/th23_cu118 \
```

blegh sorry @alexbarghi-nv , I'd missed this when reviewing before... to get the `dgl` packages you want, I think you'll also need to add this channel to the `rapids-dependency-file-generator` call (as it was added in a `--channel` arg in the `rapids-mamba-retry install` below that we're removing.

Otherwise, I think this `dgl` requirement won't be met:

https://github.com/rapidsai/cugraph-gnn/blob/87455cfedcc6721f24c783ba555af14a9a180624/conda/recipes/cugraph-dgl/meta.yaml#L28

Because the only `dgl` that `mamba` will find is from conda-forge, which doesn't have `dgl>=2.4.0` or versions with stuff like `.th23`

<img width="630" alt="image" src="https://github.com/user-attachments/assets/334528b1-5b2f-411c-96ed-013cdbeb393c" />

https://anaconda.org/conda-forge/dgl/files